src/code/z_kankyo.c

```diff
@@ -732,13 +1681,125 @@ void func_800F8970(void) {
     }
 }
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_kankyo/Environment_UpdatePostmanEvents.s")
+void Environment_UpdatePostmanEvents(PlayState* play) {
+    u8 v1;
+    u8 temp1;
+    u16 temp_a2_2;
+
+    //! FAKE: excess temp usage?
+    temp1 = (u8)((void)0, ((gSaveContext.eventInf[7] & 0xE0) >> 5));
```

Comment:
```suggestion
    temp1 = (gSaveContext.eventInf[7] & 0xE0) >> 5;
```
Can remove the `(void)0` and cast here as well.